src/app/pages/HomePage/ParaTimeSelector/Graph/index.tsx

```diff
@@ -133,15 +134,51 @@ const GraphCmp: FC<GraphProps> = ({
         fill="white"
       />
-      <circle cx="235.407" cy="36.3673" r="11.8332" transform="rotate(-90 235.407 36.3673)" fill="#01F1E3" />
-      <circle cx="235.335" cy="36.4385" r="8.02878" transform="rotate(-90 235.335 36.4385)" fill="#06152B" />
+      <circle
+        id={`${GraphEndpoint.EMERALD}-tx-outer-circle`}
+        onClick={onSelectEmeraldTxs}
+        cx="235.407"
+        cy="36.3673"
+        r="11.8332"
+        transform="rotate(-90 235.407 36.3673)"
+        fill="#01F1E3"
+      />
+      <circle
+        id={`${GraphEndpoint.EMERALD}-tx-inner-circle`}
+        onClick={onSelectEmeraldTxs}
+        cx="235.335"
+        cy="36.4385"
+        r="8.02878"
+        transform="rotate(-90 235.335 36.4385)"
+        fill="#06152B"
+      />
       <path
+        id={`${GraphEndpoint.EMERALD}-tx-label`}
+        onClick={onSelectEmeraldTxs}
```

Comment:
I would group these elements and have a single onClick, e.g.
`<g id="svg-graph-${GraphEndpoint.EMERALD}-txs" onClick={onSelectEmeraldTxs}>`

or later:
`<Link to={RouteUtils..}>`